content/collections/destination-catalog/en/moloco.md

```diff
@@ -38,22 +38,75 @@ Keep these things in mind when sending events to Moloco Commerce Media (MCM):
+| `revenue_amount`           | double | N        |             |                    | The entire revenue of the event without currency (15.5 if $15.5)                                                                                                                                              | 15.5                                                                              |
+| `page_id`                  | string | N        |             |                    | Page ID should be a uniquely assigned value for each page in the app or website. Provide a string that can identify the context of the event. Any value will be acceptable if it helps identify unique pages. | - electronics <br> - categories/12312 <br> - Azd911d <br> - /classes/foo/lectures |
+| `referrer_page_id`         | string | N        |             |                    | Similar to the referrer in HTTP, this value indicates which page the user came to the current page from.                                                                                                      | https://mycommerce.com/category/furniture                                         |
+| `search_query`             | string | N        |             |                    | Query string for the search                                                                                                                                                                                   | microwave ovens                                                                   |
+| `items`                    | obj    | N        |             |                    | Items information. Details can be found below.                                                                                                                                                                |                                                                                   |
+
+## Use cases
+
+* **Real-time User Segmentation:** By sending streaming data from Amplitude, which is a comprehensive analytics platform, to Moloco, you can create real-time user segments based on various attributes and behaviors. This enables you to target specific user groups with personalized advertisements and optimize campaign performance accordingly.
+* **Dynamic Ad Creative Optimization:** By integrating Amplitude's streaming data with Moloco's machine learning engine, you can leverage real-time user insights to dynamically optimize ad creative elements. This includes dynamically adjusting ad content, images, offers, and calls to action based on user preferences and behaviors, resulting in more engaging and effective advertisements.
```

Comment:
```suggestion
* **Dynamic ad creative optimization:** By integrating Amplitude's streaming data with Moloco's machine learning engine, you can use real-time user insights to optimize ad creative elements. This includes dynamically adjusting ad content, images, offers, and calls to action based on user preferences and behaviors, resulting in more engaging and effective advertisements.
```

Reply:
We never use "leverage" when "use" will suffice (which is almost always). Also, there were too many occurrences of "dynamically."